doc/modules/language-guide/pages/about-this-guide.adoc

```diff
@@ -10,14 +10,14 @@ ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 
 The _{proglang} Programming Language Guide_ introduces key features of the general-purpose {proglang} programming language and provides examples and reference information to help you learn the nuances of the language and the practical implications of how to apply it.
 
-The {proglang} programming language is optimized for developing programs that run on the {platform} and to work with the {sdk-long-name}. 
-However, you can also write programs using {proglang} for other platforms and to run in other contexts.
-This guide attempts to strike a balance between highlighting features that are uniquely suited to running on the {IC} and features that are generally-applicable or well-suited for programs running on other platforms.
+The {proglang} programming language is optimized for developing programs that run on the {platform} and to work with the {sdk-long-name}.
+You could, in principle, also write programs using {proglang} for more traditional platforms and to run in other contexts, though support for this is currently best-effort and incomplete.
+This guide attempts to strike a balance between highlighting features that are uniquely suited to running on the {IC} and features that are generally-applicable or well-suited for programs running on all targets.
```

Comment:
I would keep "platform" here because that's the word we use in the previous sentence.

Reply:
I didn't want to imply that the IC is a platform, because, for some reasons, that's verboten.